components/server/src/main/java/com/hotels/styx/server/netty/NettyServerConfig.java

```diff
@@ -125,42 +135,84 @@ public int nioAcceptorBacklog() {
         return this.nioAcceptorBacklog;
     }
 
+    /*
+     TODO unused: https://github.com/HotelsDotCom/styx/issues/428
+     */
     public boolean tcpNoDelay() {
         return this.tcpNoDelay;
     }
 
+    /*
+     TODO unused: https://github.com/HotelsDotCom/styx/issues/428
+     */
     public boolean nioReuseAddress() {
         return this.nioReuseAddress;
     }
 
+    /**
+     * The maximum length in bytes of the initial line of an HTTP message, e.g. {@code GET http://example.org/ HTTP/1.1}.
+     *
+     * @return maximum length of initial line
+     */
     public int maxInitialLineLength() {
         return this.maxInitialLineLength;
     }
 
+    /**
+     * The maximum combined size of the HTTP headers in bytes.
+     *
+     * @return maximum combined size of headers
+     */
     public int maxHeaderSize() {
         return this.maxHeaderSize;
     }
 
+    /**
+     * The maximum size of an HTTP chunk in bytes.
+     *
+     * @return maximum chunk size
+     */
     public int maxChunkSize() {
         return this.maxChunkSize;
     }
 
+    /*
+     TODO unused: https://github.com/HotelsDotCom/styx/issues/428
+     */
     public int maxContentLength() {
         return this.maxContentLength;
     }
 
+    /**
+     * Time in milliseconds Styx Proxy Service waits for an incoming request from client.
```

Comment:
This description is bit vague. 

It would be worth clarifying that this parameter controls the amount of tolerated inactivity while the request is being received. If the client started sending the request, and then suddenly cuts it off, styx would detect this after `requestTimeoutMillis`.